README.md

```diff
@@ -833,24 +833,25 @@ distributed training.
 
 Starting in TensorFlow 1.2, there is a new system available for reading data
 into TensorFlow models: dataset iterators, as found in the **tf.contrib.data**
-module. Data iterators are flexible, easy to reason about and to manipulate, and
+module for versions <= 1.3 and **tf.data** module for versions >= 1.4. Data 
```

Comment:
We don't really need to mention `tf.contrib.data` here anymore. We have a tf-1.2 branch for tracking 1.3 and 1.2 features.